### ISSUE TITLE
list of all services (one page / a-z / ctrl+f) - easy browsing

### ISSUE THREAD

**Issue by bgmnt** [Nitrokey/dongleauth#252]
i search a specific service and do not want to klick through all categories (pain, please no). one page with all would be fine (browser search via ctrl + f and yes, why not sorted from a to z with headline hierarchy.

maybe connected to #145

**Comment by jans23** [comment]
Doesn't the main search field address what you need?

**Comment by bgmnt** [comment]
no. it's about to see, what is available, when i do not know, what i search. without open every category.
yes, my description was misunderstandable. sorry for that.
